server.js

```diff
@@ -5,6 +5,9 @@ app.use(express.static('./build/site'))
 
 app.use('/static/assets', express.static('./build/site/assets'))
 
-app.get('/', (req, res) => res.redirect('/docs/'))
```

Comment:
Why was this taken away?

Reply:
It appeared to not do anything. `/docs/` is not an existing path.